### PR TITLE
Use standard balances

### PR DESCRIPTION
**Pull request by jerry-yu**
Use substrate balances；
move force lock functions to new pallet operation;
cross pallet storage mitigating

### CODE REVIEW

**Review by AmadeusGB — DISMISSED** [review]

File: pallets/operation/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        ) -> DispatchResultWithPostInfo {
+            ensure_root(origin)?;
+            let who = T::Lookup::lookup(who)?;
+            <T::Currency as LockableCurrency<_>>::remove_lock(id, &who);
+            Self::deposit_event(Event::UnLocked(who));
+            Ok(().into())
+        }
+
+        #[pallet::weight(T::WeightInfo::force_unreserve())]
+        pub fn force_unreserve(
```
[the remark this answers is not in the document]
Comment (jerry-yu) [reply pallets/operation/src/lib.rs:123]: ok, use balances native funciton

File: pallets/operation/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+            #[pallet::compact] value: BalanceOf<T>,
+        ) -> DispatchResultWithPostInfo {
+            const FORCE_LOCK_ID: [u8; 8] = *b"forcelck";
+            let sender = ensure_signed(origin)?;
+            ensure!(
+                <LockMemberWhiteList<T>>::get().contains(&sender),
+                Error::<T>::NotLockMember
+            );
+            let who = T::Lookup::lookup(who)?;
+            <T::Currency as LockableCurrency<_>>::set_lock(
```
[the remark this answers is not in the document]
Comment (jerry-yu) [reply pallets/operation/src/lib.rs:166]: why reserve is better?

[the remark this answers is not in the document]
Comment (jerry-yu) [reply pallets/operation/src/lib.rs:166]: using reserve is concise  at coding,
using lock is flexible，as  "/// Note that you can lock more funds than a user has."

[the remark this answers is not in the document]
Comment (xcaptain) [reply pallets/operation/src/lib.rs:166]: image
https://wiki.polkadot.network/docs/build-protocol-info
Lock means some tasks like(staking, vote) are using some balance, when these tasks ends, these balance will be returned to user.
Reserve means decrease a user's free balance, so he can't use these balance.
So if we want to forbid some users, using reserve is much more suitable, but using lock is also ok, you can decide change or not.

File: pallets/micropayment/src/lib.rs [hunk 3]
```diff
@@ -529,20 +527,13 @@ pub mod pallet {
...
-                } else {
-                    balance.free -= amount;
-                }
-                return amount;
-            });
-            match result {
-                Ok(actual_amount) => actual_amount == amount,
-                _ => false,
-            }
+            T::Currency::withdraw(
```
Comment (xcaptain) [inline pallets/micropayment/src/lib.rs:530]: https://github.com/paritytech/substrate/blob/master/frame/balances/src/lib.rs#L907-L909
`withdraw` and `deposit_creating` use `try_mutate_account`, and they will decrease and increase the total issuance, looks like not very good.
In this situation I think using a lock may be more appropriate, like the https://docs.substrate.io/rustdocs/latest/pallet_vesting/index.html vesting pallet.
When a user start using micropayment channel, we some some balance, when a server `claim_payment`, he just get from the locked balance.
It's a little off-topic, but we can discuss whether to change in the future.

Comment (jerry-yu) [reply pallets/micropayment/src/lib.rs:530]: Withdrawing ，when claming payment ,then deposit the value.  The total issurance  unchanged.
But there is temporary unbalanced, maybe we should withdraw the balance to micropayment pallet-id account, then pay fee from pallet-id account

File: pallets/operation/src/benchmarking.rs [hunk 4]
```diff
@@ -65,46 +65,18 @@ benchmarks! {
...
         let _ = T::Currency::make_free_balance_be(&user_b, existential_deposit);
-    }: set_lock_members(RawOrigin::Root, vec!(user_a,user_b))
+    }: set_reserve_members(RawOrigin::Root, vec!(user_a,user_b))
     verify {
         assert_eq!(Op::<T>::lock_member_whitelist().len(), 2);
     }
 
     impl_benchmark_test_suite!(Op, crate::tests::new_test_ext(), crate::tests::Test);
 }
-
```
Comment (xcaptain) [inline pallets/operation/src/benchmarking.rs]: remove unused code.

**Review by xcaptain — APPROVED** [review]
Remove unused comment then I'm ok with that.

**Pull request merged**